src/components/teamPhotos.js

```diff
@@ -53,18 +58,99 @@ const ProfileRole = styled.span`
 `
 
 const ProfileImg = styled(Img)`
-  border-radius: 50%;
+  transition: filter ease-in 200ms;
+`
+
+const ProfileImgWrapper = styled.div`
+  position: relative;
```

Comment:
What's `position: relative` doing for you here?

Reply:
Makes it so that the `absolute` positioning on IconWrapper refers to `ProfileImgWrapper` instead of the whole page.